// @webqit/util/js/index.js

/**
 * @imports
 */
import _getProxyTarget from './getProxyTarget.js';
import _getProxyTrap from './getProxyTrap.js';
import _getType from './getType.js';
import _instanceof from './instanceof.js';
import _internals from './internals.js';
import _isArray from './isArray.js';
import _isBoolean from './isBoolean.js';
import _isClass from './isClass.js';
import _isEmpty from './isEmpty.js';
import _isFunction from './isFunction.js';
import _isNull from './isNull.js';
import _isNumber from './isNumber.js';
import _isNumeric from './isNumeric.js';
import _isObject from './isObject.js';
import _isPlainArray from './isPlainArray.js';
import _isPlainObject from './isPlainObject.js';
import _isProxy from './isProxy.js';
import _isString from './isString.js';
import _isTypeArray from './isTypeArray.js';
import _isTypeFunction from './isTypeFunction.js';
import _isTypeObject from './isTypeObject.js';
import _isUndefined from './isUndefined.js';
import _mixin from './mixin.js';
import _proxy from './proxy.js';

/**
 * @exports
 * "js" category of modules.
 */
export {
	_getProxyTarget,
	_getProxyTrap,
	_getType,
	_instanceof,
	_internals,
	_isArray,
	_isBoolean,
	_isClass,
	_isEmpty,
	_isFunction,
	_isNull,
	_isNumber,
	_isNumeric,
	_isObject,
	_isPlainArray,
	_isPlainObject,
	_isProxy,
	_isString,
	_isTypeArray,
	_isTypeFunction,
	_isTypeObject,
	_isUndefined,
	_mixin,
	_proxy,
};
